verilog/tools/syntax/export_json_examples/verible_verilog_syntax.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2017-2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import json
+from anytree import NodeMixin, LevelOrderIter
+
+from dataclasses import dataclass
+from typing import Optional, List
+from anytree import LevelOrderIter, PostOrderIter
+
+
+class Token:
```

Comment:
Is it possible to add a BUILD file with unit-tests for these python files?  This looks like functionality that we want to protect and guarantee works for users.

Reply:
done